config/locales/forms.yml

```diff
@@ -119,14 +119,14 @@ en:
           <a href="https://www.gov.uk/" target="_blank" class="govuk-link">GOV.UK website (opens in new tab)</a>. You can apply for this role even if you do not have the
           right to work in the UK.
       jobseekers_job_application_details_break_form:
-        ended_on: For example, 01 2020
+        ended_on: For example, %{date}
```

Comment:
these are all the same, maybe we can just use one like: 

`helpers.hint.date`

Reply:
Reasonable.  Updated.